### PR TITLE
Feature/7125 speedup activities research resources 5

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[Create 1 file per email and 1 file for common strings for registry translations](https://trello.com/c/WJ0Zgbzk/7209-create-1-file-per-email-and-1-file-for-common-strings-for-registry-translations)

**Comment by DanielPalafox** [comment]
[[speedup] Activities: Research resources (5)](https://trello.com/c/zjhW8Wak/7125-speedup-activities-research-resources-5)

File: src/app/cdk/panel/panels/panels.component.html [hunk 1]
```diff
@@ -23,20 +23,22 @@
           >
           <ng-container *ngIf="!currentAmount">(0)</ng-container>
         </ng-container>
-        <ng-container *ngIf="type === 'peer-review'">
+        <ng-container *ngIf="type === 'peer-review' || 'research-resources'">
```
Comment (leomendoza123) [inline src/app/cdk/panel/panels/panels.component.html:26]: Should this be like:
`type === 'peer-review' || type === 'research-resources'`

File: src/app/record/components/research-resources/research-resources.component.ts [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+          researchResource.putCode.value
+        : false
+    return response
+  }
+
+  getLink(type: string, value: string) {
+    return this._organizationsService.getLink(type, value)
+  }
+
+  isUrl(value: string) {
```
Comment (leomendoza123) [inline src/app/record/components/research-resources/research-resources.component.ts:201]: We can remove if this is never use

File: src/app/record/components/research-resources/research-resources.component.ts [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    const response =
+      researchResource && group.defaultResearchResource[0]
+        ? group[0].defaultResearchResource.putCode.value ===
+          researchResource.putCode.value
+        : false
+    return response
+  }
+
+  getLink(type: string, value: string) {
+    return this._organizationsService.getLink(type, value)
```
Comment (leomendoza123) [inline src/app/record/components/research-resources/research-resources.component.ts:198]: Seems like this is important, but never use? 
If not require we can remove the function from the service as well

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/record/components/research-resources/research-resources.component.html [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+                    let orgDisambiguatedExternalIdentifier of organizationDisambiguated.orgDisambiguatedExternalIdentifiers
+                  "
+                >
+                  {{ orgDisambiguatedExternalIdentifier.identifierType }}:
+                  <span
+                    *ngFor="
+                      let identifier of orgDisambiguatedExternalIdentifier.all
+                    "
+                  >
+                    <ng-container *ngIf="!isUrl(identifier)">
```
Comment (DanielPalafox) [inline src/app/record/components/research-resources/research-resources.component.html:212]: @leomendoza123 the method isUrl is it used here

File: src/app/record/components/research-resources/research-resources.component.html [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+                          >
+                            Organization identifiers
+                          </h3>
+                          <div>
+                            {{ organizationDisambiguatedNested.sourceType }}
+                            <a
+                              target="_blank"
+                              rel="noopener noreferrer"
+                              href="{{
+                                getLink(
```
Comment (DanielPalafox) [inline src/app/record/components/research-resources/research-resources.component.html:403]: @leomendoza123 @leomendoza123 the method getLink is it used here

File: src/app/record/components/research-resources/research-resources.component.ts (hunk 3 above)
Comment (DanielPalafox) [reply src/app/record/components/research-resources/research-resources.component.ts:198]: https://github.com/ORCID/orcid-angular/pull/736/files#r610050341

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**